integration/scenario_test.go

```diff
@@ -173,79 +131,32 @@ func startFormProcess(wg *sync.WaitGroup, numNodes int, numVotes int, proxyArray
 	formpubkey := getFormResponse.Pubkey
 	formStatus := getFormResponse.Status
 	BallotSize := getFormResponse.BallotSize
-	Chunksperballot := chunksPerBallot(BallotSize)
+	chunksPerBallot := chunksPerBallot(BallotSize)
 
```

Comment:
would it make sense to create an instance of Form here ? That would avoid keeping each useful attribute seperately, and would allow access to chunksPerBallot

Reply:
I don't know exactly how I could create a form instance

Reply:
I was just thinking of something like 
`	election := types.Form{
		Pubkey: getFormResponse.Pubkey, 
		Status: getFormResponse.Status,
		BallotSize: getFormResponse.BallotSize
	}`

That way you have all the utility functions for Forms